feature-configs/base/sctp/sctp_role.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: sctp-pool
+  labels:
+    sctp-pool: ""
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {
+          key: machineconfiguration.openshift.io/role,
+          operator: In,
+          values: [worker-sctp],
```

Comment:
machien selector needs to match worker-sctp and worker. so something like 

```
  machineConfigSelector:
    matchExpressions:
      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,worker-sctp]}
```

if that fixes the deploy issue you're having, then we have another issue, and that's that each machine config pool needs to be aware of all possible labels